### DISCUSSION TITLE
For the picture given in the lora model, how to obtain which checkpoint model is used through the interface?

### DISCUSSION THREAD

**Discussion by zhouwanbin** [civitai/civitai#587] [Q&A]
For the picture given in the lora model, how to obtain which checkpoint model is used through the interface?

**Comment by PeterNjeim** [comment]
Click the information button in the bottom right corner of an image (letter `i` with a circle around it). If the uploader did not remove some metadata, it should show the model in there. If the information button is absent, it either means that the uploader removed the metadata, or Civitai didn't recognize the metadata. To check, download the image (right click > save image), then check the metadata. There are lots of ways to do this but for simplicity sake just use https://sdimage.info to see if there's any metadata. You might not find the model name, but you will probably see the model hash. Civitai supports searching by model hash, so search it in Civitai and if the model is available on Civitai it'll show up. If all else fails, I can help you if you share it
